docs/src/main/asciidoc/security-getting-started.adoc

```diff
@@ -3,70 +3,74 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= Getting Started with Quarkus Security
-
+= Secure a Quarkus application with basic authentication
```

Comment:
```suggestion
= Secure a Quarkus application with Basic authentication
```

Reply:
`Basic` authentication is a well known concept, so while it may not match Red Hat guidelines, IMHO we need to retain a capital `B`

Reply:
Thanks, @sberyozkin - Apologies for more churn on this!

If the content is specifically referring to the HTTP RFC specification for the 'Basic' authentication scheme (https://www.rfc-editor.org/rfc/rfc7617.txt), as I think it is here, then I totally agree.  I will follow up with Red Hat style guidelines to get this variation and clarification added to our style guide.
 
For other references, if the content is more generally referring to basic authentication as a common operation that does not formulate a unique product or feature identity, it does not deserve proper noun treatment and lowercase '_basic authentication_' is more appropriate, as per Red Hat guidelines and what we see commonly used in other non-Red Hat docs.

(CC-ing FYI: @inoxx03,  @sheilamjones, @hmanwani-rh, @MichalMaler )